src/components/ReportActionItem/MoneyRequestView.tsx

```diff
@@ -340,6 +340,9 @@ function MoneyRequestView({
         if (formattedOriginalAmount) {
             amountDescription += ` ${CONST.DOT_SEPARATOR} ${translate('iou.original')} ${formattedOriginalAmount}`;
         }
+        if (getOriginalTransactionWithSplitInfo(transaction).isExpenseSplit) {
```

Comment:
@ryntgh I'm sorry, but we should avoid using `getOriginalTransactionWithSplitInfo` since it uses `allTransactions` which is retrieved via Onyx.connect and we're deprecating uses of Onyx.connect, so this PR will make deprecating it harder. Can we instead subscribe to the `originalTransactionID` via useOnyx and retrieving the data we need that way?